src/process_utility.c

```diff
@@ -4496,7 +4557,9 @@ process_altertable_end_table(Node *parsetree, CollectedCommand *cmd)
 		switch (cmd->type)
 		{
 			case SCT_Simple:
-				process_altertable_end_simple_cmd(ht, cmd);
+				process_altertable_end_subcmd(ht,
```

Comment:
Would be good to add coverage for this as well.

Reply:
this is unrelated refactoring maybe i should move into separate PR